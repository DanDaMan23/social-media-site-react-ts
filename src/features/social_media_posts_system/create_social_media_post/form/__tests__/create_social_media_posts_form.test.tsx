import { render } from "@testing-library/react"
import CreateSocialMedialPostForm from "../create_social_media_post_form"

describe("create_social_media_posts_form test", () => {
  it("should render correctly", () => {
    const { container } = render(
      <CreateSocialMedialPostForm register={jest.fn()} />
    )

    expect(container).toMatchSnapshot()
  })
})
