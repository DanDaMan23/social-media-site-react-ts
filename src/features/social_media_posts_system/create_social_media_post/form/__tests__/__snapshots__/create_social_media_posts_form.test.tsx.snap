// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create_social_media_posts_form test should render correctly 1`] = `
<div>
  <form
    class=""
  >
    <div
      class="mb-3"
    >
      <textarea
        class="form-control"
        id="postContent"
        rows="10"
      />
    </div>
  </form>
</div>
`;
